### ISSUE TITLE
Triple loss

### ISSUE THREAD

**Issue by MrGreensushi** [MrGreensushi/MachineLearningProject#4]
Implementare classe su python per il triple loss

1. creare triplette di immagini (1 modello, 1 esempio positivo, 1 esempio negativo)
2. il modello prendo in ingresso la tripletta e in paralello lo manda a 3 CNN
3. Distanza euclidea dei 3 cnn

Scelta iniziale degli iperparametri
- Funzione di loss e il valore alpha associato
- numero epoche
- early stop -> parametro da analizzare: https://www.tensorflow.org/api_docs/python/tf/keras/metrics/TopKCategoricalAccuracy
 
plottare topk accuracy alla fine
